### PR TITLE
Implement logo-card component

### PR DESCRIPTION
**Pull request by gdostie**
Implemented `logo-card` component

screenshot from 2018-03-09 15-02-08

### CODE REVIEW

File: scss/variables.scss [hunk 1]
```diff
@@ -451,3 +451,9 @@ $ribbon-x-offset: calc((#{$ribbon-width} / -2) + #{$ribbon-distance-from-corner}
 $ribbon-font-size: 0.8em;
 $ribbon-letter-spacing: 1px;
 $ribbon-rotation-angle: 45deg;
+
+// Logo card
+$logo-card-title-color: #4a4a4a;
```
Comment (GermainBergeron) [inline scss/variables.scss]: is this supposed to be `$dark-grey`?

**Comment by GermainBergeron** [comment]
Looks really good!

**Comment by gdostie** [comment]
New logo-card-logo margin-right :
image

File: scss/variables.scss (hunk 1 above)
Comment (gdostie) [reply scss/variables.scss]: `$dark-grey` is #373737, the new color is took from the mockup

**Pull request merged**